CONTRIBUTORS.txt

```diff
@@ -60,27 +68,30 @@ needs to be listed here.
 - John Connolly <connoljo2@gmail.com>
 - John Holdsworth <github@johnholdsworth.com>
 - John Lin <johnlinvc@gmail.com>
+- Joshua Rutkowski <54906829+joshrutkowski@users.noreply.github.com>
```

Comment:
@joshrutkowski if you'd like a proper email address here, please let me know which.

Reply:
jr.rtkski@gmail.com

Reply:
thanks, updated.